rasa/test.py

```diff
@@ -16,9 +17,13 @@
 logger = logging.getLogger(__name__)
 
 
-def test_core_models_in_directory(model_directory: Text, stories: Text, output: Text):
+def test_core_models_in_directory(
+    model_directory: Text, stories: Text, output: Text
+) -> None:
     from rasa.core.test import compare_models_in_dir, plot_core_results
 
+    model_directory = _test_core_models_in_directory_input(model_directory)
```

Comment:
`_test_core_models_in_directory_input` doesn't really tell what the helper function is doing / what it is used for.

How about (just a suggestion / feel free to find a better name 😀):
```suggestion
    model_directory = _get_sanitized_model_directory(model_directory)
```